.github/workflows/test.yaml

```diff
@@ -869,18 +932,21 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - uses: pnpm/action-setup@v2.2.4
+        with:
+          version: 7
+
       - uses: actions/setup-node@v3
         with:
-          node-version: 14
-          cache: 'yarn'
-          # Only cache the root node_modules
-          cache-dependency-path: 'yarn.lock'
+          node-version: 16
+          cache: ${{ contains(matrix.platform, 'redwood') && 'yarn' || 'pnpm' }}
+          cache-dependency-path: ${{ github.job }}/${{ matrix.platform }}/${{ contains(matrix.platform, 'redwood') && 'yarn.lock' || 'pnpm-lock.yaml' }}
 
       - name: Define Client Engine Type to test
         run: echo "PRISMA_CLIENT_ENGINE_TYPE=${{ matrix.clientEngine }}" >> $GITHUB_ENV
 
       - name: Install Dependencies
-        run: yarn install
+        run: ${{ contains(matrix.platform, 'redwood') && 'yarn' || 'pnpm install' }}
```

Comment:
?

Reply:
Similar to https://github.com/prisma/ecosystem-tests/pull/3463#discussion_r1159861542

Reply:
Redwood is stuck on yarn so we have to run it with it